src/gridstack-poly.js

```diff
@@ -5,6 +5,18 @@
  * gridstack.js may be freely distributed under the MIT license.
 */
 
+// https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent/CustomEvent

```

Comment:
site has it wrapped in a function it calls.. maybe to scope the CustomEvent var ? I would do the same please.

Reply:
My mistake. Thought the whole poly file was scoped for reason. Updated.